subsys/bluetooth/host/iso.c

```diff
@@ -1915,6 +1955,21 @@ int bt_iso_big_create(struct bt_le_ext_adv *padv, struct bt_iso_big_create_param
 	return err;
 }
 
+static void store_bis_broadcaster_info(const struct bt_hci_evt_le_big_complete *evt,
+				       struct bt_iso_broadcaster_info *info)
+{
+	info->sync_delay = sys_get_le24(evt->sync_delay);
+	info->latency = sys_get_le24(evt->latency);
+	info->phy = evt->phy;
+	info->max_subevent = evt->nse;
+	info->bn = evt->bn;
+	info->irc = evt->irc;
+	info->interval = sys_le16_to_cpu(evt->iso_interval);
+	/* Transform to ms */
+	info->pto = info->interval * evt->pto;
```

Comment:
for intervals of x.25, x.5 and x.75ms this calculation multiplies the error if the unit of `evt->iso_interval` is ms - but I think the unit of that is slot-pairs (1.25ms). hence you need to multiply by 1.25ms to get ms. Maybe micro seconds is a better unit to use?

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/45077#discussion_r857551073